Source/Engine/Graphics/Models/Mesh.cpp

```diff
@@ -403,6 +404,9 @@ void Mesh::Draw(const RenderContext& renderContext, MaterialBase* material, cons
 
 void Mesh::Draw(const RenderContext& renderContext, const DrawInfo& info, float lodDitherFactor) const
 {
+    if (!IsInitialized())
```

Comment:
It already checks for initialized state a few lines below.

Reply:
My bad, how did I miss that.